### DISCUSSION TITLE
3D fences

### DISCUSSION THREAD

**Discussion by AlexValue** [organicmaps/organicmaps#2710] [Ideas]
I want to propose an idea for a new display of fences on the map. They can be drawn in 3D (like buildings). I have attached a screenshot with an example. This would make it easier for new users to navigate and overall make the map prettier.

![Screenshot_2022-06-11-13-03-51-819_ru yandex yandexmaps~2](https://user-images.githubusercontent.com/107300467/173181194-d8119ff6-e788-44f6-bf3f-386e96ff3cb3.jpg)

![Screenshot_2022-06-11-13-52-43-944_ru yandex yandexmaps~2](https://user-images.githubusercontent.com/107300467/173181220-a86ade17-07a3-44fe-ab9c-95756e204812.jpg)

![Screenshot_2022-06-11-13-53-19-931_ru yandex yandexmaps~2](https://user-images.githubusercontent.com/107300467/173181228-82fb3927-85e5-4246-ba5d-d64aa972452f.jpg)

I would also like to have 2 display options: a regular fence and a low fence. By low fence I mean fences that can be stepped over (below 90 centimeters). Of course, if opportunities allow OpenStreetMaps.

**Comment by pastk** [comment]
It had been suggested before in #2275 and #1469
